projectDocs/dev/buildingNVDA.md

```diff
@@ -18,24 +20,31 @@ scons source user_docs
 
 While simply testing or committing changes, it may be faster usually just doing `scons source` as user documentation will change each time the revision number changes.
 
-You can speed up scons calls by appending the `--all-cores` parameter to the call, however note that output will be scrambled:
+You can speed up scons calls by appending the `--all-cores` or `-j N` (where `N` is the number of cores to use) parameter to the call, however note that this can cause errors, and output will be scrambled:
 
 ```cmd
 scons source --all-cores
 ```
 
+If you are experiencing errors building NVDA with threading enabled, please force a serial build with the `-j 1` parameter.
+This will make tracking down the issue much easier, and may even resolve it.
+
 ## Running the Source Code
+
 It is possible to run NVDA directly from source without having to build the full binary package and launcher.
 To launch NVDA from source, using `cmd.exe`, execute `runnvda.bat` in the root of the repository.
 
 To view help on the arguments that NVDA will accept, use the `-h` or `--help` option.
 These arguments are also documented in the [user guide](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#CommandLineOptions).
 
-## Building NVDA
```

Comment:
is this anchor used anywhere?

Reply:
I searched the repo for `buildingNVDA.md#building-nvda` and didn't come up with anything. Note that its implicit slug would clash with the top heading though anyway.